src/ctest.c

```diff
@@ -232,7 +235,7 @@ size_t RunTests(const TEST_FUNCTION_DATA* testListHead, const char* testSuiteNam
             /*revert console to initial state*/
             if (!SetConsoleMode(std_out_handle, console_mode_initial))
             {
-                LogWarning("Error resetting console mode to initial value of %" PRIx32 ". GetLastError()=%" PRIx32 "", console_mode_initial, GetLastError());
```

Comment:
>PRIx32 [](start = 80, length = 6)

probably still PRIx32 (like above)

Reply:
Fixed.

---
In reply to: [561430305](https://github.com/Azure/ctest/pull/74#discussion_r561430305) [](ancestors = 561430305)